ui/tests/test_url_redirects.py

```diff
@@ -268,6 +268,258 @@ def test_article_redirects_query_params(url, expected_pattern, client):
         'https://trade.great.gov.uk/'
         'campaign/legal-is-great/singapore/'
     )),
+    ('/help/contact/', reverse('contact-us-interstitial-service-agnostic')),
```

Comment:
these redirects support the historic `help` app redirects.

the assumption here is the current `help` app will redirect all requests to `great.gov.uk/help/<original_path>`